### DISCUSSION TITLE
[ERLEDIGT] Bestückte openDTU_wDisplay2 Breakoutboards abzugeben

### DISCUSSION THREAD

**Discussion by BabyFnord** [tbnobody/OpenDTU#903] [Show and tell]
Moinsen,

`Vorab: Falls die Aktion unerwünscht ist, bitte Bescheid geben und ich putze das wieder weg. `

vom [openDTU_wDisplay2-Breakoutboard](https://github.com/SteffMUC/openDTU_wDisplay2) habe ich **keine** bestückten PCBs mehr über (→ Pin Header + Kondensator). Eins reichte mir, weil aber die Mindestbestellmenge stets fünf ist und es anderen helfen mag, hatte ich den restlichen Kram mitbestellt und aufgebaut. 

![OpenDTU Breakoutboards 1](https://user-images.githubusercontent.com/39904614/237039476-d307c78e-cb8c-485e-a274-87dc611e185c.jpg)
![OpenDTU Breakoutboard fertig 1](https://user-images.githubusercontent.com/39904614/237039508-e92a36fb-964f-4037-931d-69df4b2eaaab.jpg)

~~Abgabe zum `Selbstkostenpreis`, Zahlung per PayPal Freunde & Familie oder Banküberweisung:
- €12 inkl. Versand per Deutsche Post Maxibrief (DE) mit Einwurfeinschreiben.
- €7 bei Abholung~~

Für eine komplette OpenDTU benötigt ihr noch (*Bay-Links als Beispiel – wer Geduld hat, kann direkt in CN mit 1 Monat ++ Wartezeit bestellen):
- [ ]  ESP32S NodeMCU (38 Pins) https://www.ebay.de/itm/122795614934 
- [ ]  Funkmodul NRF24L01+ https://www.ebay.de/itm/255283312809
- [ ]  OLED Display SSD1306 https://www.ebay.de/itm/153900883604
 

### Plug'n'Play-Kurzbeschreibung
- [ ]  Zuerst die [Firmware](https://github.com/tbnobody/OpenDTU/releases) auf die _noch nicht verbaute_ ESP32S NodeMCU per [Browser (Chromium/Chrome/Firefox) und Webflasher](https://espressif.github.io/esptool-js/) [aufspielen (auf die richtige Zuordnung der .bin Dateien achten)](https://github.com/tbnobody/OpenDTU#using-the-pre-compiled-bin-files). 
- [ ]  Nach der [Ersteinrichtung](https://github.com/tbnobody/OpenDTU#first-configuration) zunächst das Geräteprofil [nodemcu_esp32.json](https://github.com/tbnobody/OpenDTU/blob/master/docs/DeviceProfiles/nodemcu_esp32.json) aus dem OpenDTU Repo laden, und als Profil „Generic NodeMCU 32 with SSD1306“ auswählen. Unter `Hardware-Einstellungen → Display → Rotation` bei Bedarf `180°` auswählen. Sofern das Geräteprofil nicht aktiv wird (Display geht nicht bzw. keine Funkverbindung zum Inverter), kann ein Power-Cycle helfen (Stromversorgung der DTU für ein paar Sekunden unterbrechen).
 

Mit dem Breakoutboard ist die Sache in unter 10' erledigt, _sans_ Löten/Gefummel mit Kabeln. 
Wer's braucht, findet das passende Gehäuse zum Selberdrucken auf der gleichen Projektseite [openDTU_wDisplay2-Breakoutboard](https://github.com/SteffMUC/openDTU_wDisplay2).

Ciao, BF

**Comment by chanzler** [comment]
Hi, ich würde eines nehmen. Kannst du mir dazu eine PN schicken?

**Reply by BabyFnord** [reply]
Es gibt keine PN-Funktion auf Github, ich schreibe dir an deine im Profil hinterlegte Mailadresse 😉

**Reply by BabyFnord** [reply]
1. Breakoutboard ist in Zustellung.

**Comment by quaternionma** [comment]
Hi, würde auch eins nehmen. Einfach per Mail anpingen ;-)

**Reply by BabyFnord** [reply]
Ping!

**Reply by BabyFnord** [reply]
2. Breakoutboard ist in Zustellung.

**Comment by carlothefirst** [comment]
Hallo,
hätte Interesse.
Student
Kontakaufnahme?

**Reply by BabyFnord** [reply]
Fein. Was aber ist eine `Student Kontaktaufnahme`? Schreib hier deine Mail rein, ich schreibe dich an, und anschließend kannste die hier wieder löschen.

**Comment by carlothefirst** [comment]


**Reply by BabyFnord** [reply]
@carlothefirst Da GitHub Mailadressen anscheinend anonymisiert, kannst du mich nun über meine Profilseite anschreiben.

**Reply by BabyFnord** [reply]
Dein Breakoutboard ist jetzt auch in Zustellung … damit kann hier jetzt zu 🙂.

**Reply by carlothefirst** [reply]
gibt es dafür einen Gehäusevorschlag?

**Reply by BabyFnord** [reply]
> Wer's braucht, findet das passende Gehäuse zum Selberdrucken auf der gleichen Projektseite [openDTU_wDisplay2-Breakoutboard](https://github.com/SteffMUC/openDTU_wDisplay2).

😉

**Comment by BabyFnord** [comment]
Das 3. Breakoutboard ist in Zustellung. ~~Eins ist noch verfügbar~~ … 💁🏻‍♂️
